src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/services/DomainEventListener.kt

```diff
@@ -16,18 +16,12 @@ import java.util.concurrent.CompletableFuture
 
 const val RETRY_CREATE_MAPPING = "RETRY_CREATE_MAPPING"
 
-abstract class DomainEventListener(
-  internal val service: CreateMappingRetryable,
+abstract class DomainEventListenerNoMapping(
```

Comment:
Needed to split out the code for mapping retries into a subclass because the prison person sync doesn't have a mapping table (using common identifier of prisonerNumber).